packages/fxa-auth-server/lib/routes/utils/subscriptions.js

```diff
@@ -24,45 +18,33 @@ const SubscriptionUtils = (module.exports = {
     ...plan.plan_metadata,
   }),
 
+  // Parse a comma-separated list of capabilities with allowance for varied whitespace
+  splitCapabilities: s =>
```

Comment:
Added some defensive coding here, as I tried entering a Stripe metadata property with `cap1, cap2` (space after the comma) and got a ` cap2` with a leading space for capability.